_config.yml

```diff
@@ -4,5 +4,14 @@ scalaversion: "2.12.0"
 
 baseurl: ""
 markdown: kramdown
-exclude: [bundle-vendor]
+exclude: [
+  bundle-vendor,
+  blog/README.md,
+  training/README.md,
+  README.md,
+  news/README.md,
+  events/README.md,
+  Gemfile,
+  Gemfile.lock
+  ]
```

Comment:
`*.md` knocks out nearly the whole site. I get the your point though. Your version is cleaner.